__tests__/components/App.test.js

```diff
@@ -1,6 +1,7 @@
 // Copyright 2019 Stanford University see LICENSE for license
 
 import 'jsdom-global/register'
+import 'isomorphic-fetch'
```

Comment:
why do we need this import now?

Reply:
I don't really know why.  If we omit it, we get "fetch is not defined".